packages/drawer/src/types.tsx

```diff
@@ -27,7 +27,7 @@ export type DrawerNavigationConfig<T = DrawerContentOptions> = {
    * - `back`: The drawer is revealed behind the screen on swipe.
    * - `slide`: Both the screen and the drawer slide on swipe to reveal the drawer.
    */
-  drawerType?: 'front' | 'back' | 'slide';
+  drawerType?: 'front' | 'back' | 'slide' | 'sidebar';
```

Comment:
Let's change to `'sidebar'` to `'permanent'` according to @EvanBacon's suggestion

```suggestion
  drawerType?: 'front' | 'back' | 'slide' | 'permanent';
```